### PR TITLE
refactored module01

### PR DESCRIPTION
**Pull request by 3rd-Son**
This PR contains a refactored version of module01
Removed unnecessary information and added important ones

### CODE REVIEW

File: courses/GENAI/Module_01_Introduction_to_Swarmauri/Notebook_02_SDK_Installation_Setup.ipynb
Comment (cobycloud) [inline courses/GENAI/Module_01_Introduction_to_Swarmauri/Notebook_02_SDK_Installation_Setup.ipynb:1]: please also show how to add the env as a juypter kernel

File: courses/GENAI/Module_01_Introduction_to_Swarmauri/Notebook_03_Features_of_Swarmauri.ipynb
Comment (cobycloud) [inline courses/GENAI/Module_01_Introduction_to_Swarmauri/Notebook_03_Features_of_Swarmauri.ipynb:1]: explain the general pattern, for example:

"to import you can use `from swarmauri.<resource_kind>.concrete import <subclass type>`

something along those lines, so they understand generally how to navigate. Explain that we provide many subtypes and that the few shown here are just examples

**Review by cobycloud — CHANGES_REQUESTED** [review]
small changes, but overall a good refresh

File: courses/GENAI/Module_01_Introduction_to_Swarmauri/Notebook_03_Features_of_Swarmauri.ipynb
Comment (3rd-Son) [reply courses/GENAI/Module_01_Introduction_to_Swarmauri/Notebook_03_Features_of_Swarmauri.ipynb:1]: On it

**Comment by 3rd-Son** [comment]
@cobycloud 
All changes have been made

**Review by cobycloud — APPROVED** [review]

**Pull request merged**